thirdparty/mbedtls/include/godot_module_mbedtls_config.h

```diff
@@ -49,6 +49,13 @@
 #undef MBEDTLS_DES_C
 #undef MBEDTLS_DHM_C
 
+#ifdef THREADS_ENABLED
+// In mbedTLS 3, the PSA subsystem has an implicit shared context, MBEDTLS_THREADING_C is required to make it thread safe.
```

Comment:
Did you test how it behaves with `threads=no`?

Reply:
I did not, but if threads are disabled, there can be no concurrency issue, so we wouldn't need that.

We might want to build with threads=no just to see if some of the new code spilled there, in theory, that build should be unchanged.